doc/source/ray-core/scheduling/placement-group.rst

```diff
@@ -309,19 +309,13 @@ You can schedule actors or tasks to a placement group using
 
   When you use an actor with a placement group, always specify ``num_cpus``.
 
-  When you don't specify (e.g., ``num_cpus=0``), a placement group option is ignored,
-  and the task and actor don't use the reserved resources.
-  
-  Note that by default (with no arguments to ``ray.remote``),
+  By default (with no arguments to ``ray.remote``), a Ray actor requires 1 CPU when it is scheduled. But after it is created, it occupies 0 CPUs.
```

Comment:
I think this can be clearer. Which of these two options is it?

1. The node where the actor is scheduled has at least 1 CPU, and the CPU is not in use at the time of scheduling.
2. The node where the actor is scheduled has at least 1 CPU, but it could be in use by another task or actor at the time of scheduling.

Also, it'd be useful to say a little bit about why we do this because it sounds a bit strange. I actually didn't know that :D

Reply:
updated, PTAL.